src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
First the github reference of the array of signal is wrong, SEGV is not between USR1 and USR2, this is really a joke and the fact you did not see if strange to me.
Secondly, signal does not exist on windows, cygwin library abstract it, windows receive messages and system are manipulated through another favility specific to window

Hence the all utils code surrounding signal is a complete missleading one.

For you reference this is a legit array of signal name

const char *const sys_signame[NSIG] = {
        "Signal 0",
        "HUP",          /* SIGHUP */
        "INT",          /* SIGINT */
        "QUIT",         /* SIGQUIT */
        "ILL",          /* SIGILL */
        "TRAP",         /* SIGTRAP */
        "ABRT",         /* SIGABRT */
        "EMT",          /* SIGEMT */
        "FPE",          /* SIGFPE */
        "KILL",         /* SIGKILL */
        "BUS",          /* SIGBUS */
        "SEGV",         /* SIGSEGV */
        "SYS",          /* SIGSYS */
        "PIPE",         /* SIGPIPE */
        "ALRM",         /* SIGALRM */
        "TERM",         /* SIGTERM */
        "URG",          /* SIGURG */
        "STOP",         /* SIGSTOP */
        "TSTP",         /* SIGTSTP */
        "CONT",         /* SIGCONT */
        "CHLD",         /* SIGCHLD */
        "TTIN",         /* SIGTTIN */
        "TTOU",         /* SIGTTOU */
        "IO",           /* SIGIO */
        "XCPU",         /* SIGXCPU */
        "XFSZ",         /* SIGXFSZ */
        "VTALRM",       /* SIGVTALRM */
        "PROF",         /* SIGPROF */
        "WINCH",        /* SIGWINCH */
        "INFO",         /* SIGINFO */
        "USR1",         /* SIGUSR1 */
        "USR2",         /* SIGUSR2 */
        "THR",          /* SIGTHR */
};